lib/index.js

```diff
@@ -62,12 +74,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       store: ['validate', (results, callback) => {
         // generate new random ID for inbox
         const inbox = results.validate;
-        inbox.id = routes.inboxes + '/' + uuidV4();
-        const { owner: owner=req.user.identity.id } = inbox;
```

Comment:
We've been putting spaces before var names when using this destructuring declaration style as it seems to be what others do. It's new, so we haven't settled on anything yet and can discuss this if you prefer no spaces.